### PR TITLE
Statistics calculator

### PR DESCRIPTION
**Pull request by govardhanshah456**
### Issue Id you have worked upon - 
#18 

### Briefly explain your program logic - 
Mostly, it was just basic mathematics, only challenging thing was handling float Numbers, I did that using parseFloat and when I had to display the answer, I used toFixed method to show output upto certain decimal places.

### Screenshots(Attach 2 screenshots of your own input and output) - 
image
image
image
<br>

### By raising this PR I affirm that -
- [x] My code follows the guidelines of this project.<br>
- [x] I have performed a self-review of my own code.<br>
- [x] I have commented my code.<br>
- [x] My code gives the correct output.<br>

- [x] I affirm that I strictly follow contributing guidelines and code of conduct.

### CODE REVIEW

**Comment by SarthakKeshari** [comment]
@govardhanshah456, A good implementation. But kindly make your contribution to the folder path mentioned in the issue comments. Home.js and App.js have already been configured for a smooth contribution. Kindly do the needful.

**Comment by govardhanshah456** [comment]
@SarthakKeshari Hey,you can check now.

File: calculators/src/App.js
Comment (SarthakKeshari) [inline calculators/src/App.js:1]: Kindly revert back the changes in this file.

**Comment by SarthakKeshari** [comment]
@govardhanshah456, Kindly check the comments above

File: calculators/src/App.js
Comment (govardhanshah456) [reply calculators/src/App.js:1]: Hey, by changes which ones o you mean? i see a new line and space change only for this file in the mentioned commit. is that it?

Comment (govardhanshah456) [reply calculators/src/App.js:1]: i have done the changes which i saw in that file, if there is any other change left, please let me know.

**Comment by SarthakKeshari** [comment]
@govardhanshah456, Kindly revert Home.js too

**Comment by govardhanshah456** [comment]
@SarthakKeshari reverted Home.js to its Original state

**Review by SarthakKeshari — APPROVED** [review]

**Comment by SarthakKeshari** [comment]
Congratulations 🎉 @govardhanshah456 on your contribution.

**Comment by SarthakKeshari** [comment]
@govardhanshah456, 
Really impressive work. 5-appreciation points to this contribution. 
Looking forward for more contributions.

**Comment by govardhanshah456** [comment]
> @govardhanshah456, Really impressive work. 5-appreciation points to this contribution. Looking forward for more contributions.

Yeah, thank you very much, for sure.

**Comment by SarthakKeshari** [comment]
### Hey 👋, may you also encourage us by clicking the star icon on the repository main page. ⭐
image

**Pull request merged**